compiler/src/iree/compiler/Codegen/Common/GPU/GPUMaterializeEncoding.cpp

```diff
@@ -301,13 +320,25 @@ struct GPUSetEncodingOpLoweringConversion
 
 } // namespace
 
+// TODO(hanchung): Remove the wrapper after allowing the type converter to carry
+// the targetAttr. For now, follow what CPU is doing.
+static MaterializeEncodingFn
+getMaterializeEncodingFn(IREE::HAL::ExecutableTargetAttr targetAttr) {
+  return
+      [targetAttr](
+          RankedTensorType tensorType) -> FailureOr<MaterializeEncodingInfo> {
+        return materializeEncodingForTarget(tensorType, targetAttr);
+      };
+}
+
```

Comment:
This TODO will be killed after we land https://github.com/iree-org/iree/pull/18242